app/templates/components/ui-table/cell/events/view/tickets/orders/cell-amount.hbs

```diff
@@ -0,0 +1 @@
+US$ {{number-format record.amount}}
```

Comment:
@geekyd Won't that seem counter intuitive ? To use an admin/sales/ template, maybe we can let this be and later in a refactor take care of the amount fields used  everywhere